src/SwipeRating.js

```diff
@@ -103,6 +112,14 @@ export default class SwipeRating extends Component {
     }
   }
 
+  handleLayoutChange() {
```

Comment:
why is this required?

Reply:
The problem was any swipe events are started from the center of the swipe view. This is because we only changed swipe value depending on the dx value from the center of it. I decided to add the relative distance from the center of the swipe view to solve this problem. (center + relativeX + dx)

![image](https://user-images.githubusercontent.com/65849802/111990216-4241f800-8b56-11eb-8ec1-52886a5fac6c.png)
There is only x0 which is the screen coordinates of the responder grant. This is absolute position of screen, and we need the tap event's relative position of the swipe component view 's center to calculate the value. So this function is called in the swiper view's `onLayout` event.